src/components/layout/sidebar-steps.tsx

```diff
@@ -5,22 +5,22 @@ import { StepItem } from '../ui/step-item.tsx'
 import { StepItemContainer } from '../ui/step-item-container.tsx'
 
 export default function SidebarSteps() {
-  const { car, ipni, filecoinPin } = SIDEBAR_CONFIG.documentation
+  const { car, ipni, filecoinPay, serviceProvider, pieceCid, standardIpfsTooling, filecoinPinAddBehindTheScenes } = SIDEBAR_CONFIG.documentation
 
   return (
     <StepItemContainer>
       <StepItem step={1}>
-        <p>A FIL wallet, a funded Filecoin Pay Account, and Warm Storage Service have been set up automatically.</p>
+        <p>A FIL wallet, a funded <TextLink href={filecoinPay}>Filecoin Pay</TextLink> Account, and <TextLink href={filecoinPay}>Warm Storage Service</TextLink> have been set up automatically.</p>
```

Comment:
are these supposed to be the same link?
```suggestion
        <p>A FIL wallet, a funded <TextLink href={filecoinPay}>Filecoin Pay</TextLink> Account, and <TextLink href={warmStorage??}>Warm Storage Service</TextLink> have been set up automatically.</p>
```

Reply:
fixed - thanks